docs/user-guide/timeouts/timeouts-main.md

```diff
@@ -0,0 +1,331 @@
+# Introduction
+
+In the following, we will give a basic classification of timeouts, explain some
+candidate causes for timeouts, and show ways to sometimes prevent them. We give
+a glimpse into the more theoretical background of timeouts in program
+verification in [this section](timeouts-theory.md)
+
+We classify Certora Prover timeouts as follows:
+1.  Timeouts that happen before SMT solvers are running 
```

Comment:
`.` at the end or do not start with a `T` (but `t`). ditto below

Reply:
alright, changed case (never sure about these things .. will try to remember)